sklearn/model_selection/_split.py

```diff
@@ -994,16 +994,16 @@ class ShuffleSplit(BaseShuffleSplit):
 
     Parameters
     ----------
-    n_splits : int (default 10)
+    n_splits : int, default 10
         Number of re-shuffling & splitting iterations.
 
-    test_size : float, int, or None, default 0.1
-        If float, should be between 0.0 and 1.0 and represent the
-        proportion of the dataset to include in the test split. If
-        int, represents the absolute number of test samples. If None,
-        the value is automatically set to the complement of the train size.
+    test_size : float, int, None, or 'default', default 'default'
```

Comment:
we'd just say "optional" and describe the default behaviour below. We don't really want users explicitly specifying 'default'.